openssl-dynamic/src/main/c/sslcontext.c

```diff
@@ -1027,7 +942,7 @@ TCN_IMPLEMENT_CALL(void, SSLContext, setNpnProtos)(TCN_STDARGS, jlong ctx, jobje
     }
 }
 
-TCN_IMPLEMENT_CALL(void, SSLContext, setAlpnProtos)(TCN_STDARGS, jlong ctx, jobjectArray alpn_protos,
+TCN_IMPLEMENT_CALL(void, SSLContext, setAlpnProtos0)(TCN_STDARGS, jlong ctx, jobjectArray alpn_protos,
```

Comment:
```suggestion
TCN_IMPLEMENT_CALL(void, SSLContext, setAlpnProtos0)(TCN_STDARGS, jlong ctx, jbyteArray alpn_protos,
```

Reply:
Also, how come this isn't breaking any builds?

Reply:
Because we preserve the same method signature https://github.com/netty/netty-tcnative/pull/610/files#diff-59b5a3b40a7b1fefad13264deca0c0ac7bf98a5810666fc88114b9b2e49e6c63R566